src/server/editorServices.ts

```diff
@@ -1718,7 +1997,8 @@ namespace ts.server {
             const rootFiles: protocol.ExternalFile[] = [];
             for (const file of proj.rootFiles) {
                 const normalized = toNormalizedPath(file.fileName);
-                if (getBaseFileName(normalized) === "tsconfig.json") {
+                const baseFileName = getBaseFileName(normalized);
+                if (baseFileName === "tsconfig.json" || baseFileName === "jsconfig.json") {

```

Comment:
There should be a helper function for testing whether a filename looks like a config file; I think this is done in a few other places too.